src/main/src/main/java/org/vfny/geoserver/util/DataStoreUtils.java

```diff
@@ -320,23 +320,21 @@ else if (!resourceInfo.getStore().getType().equalsIgnoreCase("Web Feature Server
 
     // A utility method for retreiving supported SRS on WMTSLayerInfo resource
     public static List<String> getOtherSRSFromWMTSStore(WMTSLayerInfo wmtsLayerInfo) {
+        List<String> otherSRS = Collections.EMPTY_LIST;
```

Comment:
```suggestion
        List<String> otherSRS = Collections.emptyList();
```

Avoids an unchecked conversion warning (the method return type can use type inference, the constant cannot)

Reply:
thanks, fixed